packages/pay/README.md

```diff
@@ -78,3 +97,194 @@ try {
+
+One of `amountToSend` or `amountToDeliver` must be provided, or throws `PaymentError.UnknownPaymentTarget`.
+
+##### `destinationAssetCode`
+
+> _Optional_: `string`
+
+Asset code or symbol identifying the asset the recipient will receive. For example: `USD`, `JPY`, `BTC`.
+
+Required if `amountToDeliver` was also provided, or throws `PaymentError.UnknownDestinationAsset`, since the denomination of the payment must always be known in advance. For fixed source amount payments, the destination asset details will be fetched automatically from the recipient using STREAM.
+
+##### `destinationAssetScale`
+
+> _Optional_: `number`
+
+The precision of the recipient's asset denomination: the number of decimal places of the normal unit of the destination asset. For example, if the destination account is denominated in units of $0.01, the asset scale is 2, since the unit is 2 orders of magnitude smaller than 1 U.S. dollar. The asset scale must be an integer between 0 and 255.
+
+Required if `amountToDeliver` was also provided, or throws `PaymentError.UnknownDestinationAsset`.
+
+##### `slippage`
+
+> _Optional_: `number`
+
+Percentage to subtract from the external exchange rate to determine the minimum acceptable exchange rate for each packet. By default, `0.01` or 1% slippage below the external exchange rate (see below) is used.
+
+Throws `PaymentError.InvalidSlippage` if the slippage is not a number between `0` and `1`, or is `NaN`.
+
+##### `prices`
+
+> _Optional_: `{ [assetCode: string]: number }`
+
+Object of asset codes to prices in a standardized base asset to compute exchange rates. For example, here's prices using U.S. dollars as a base asset:
+
+```js
+{
+  USD: 1,
+  EUR: 1.09,
+  BTC: 8806.94
+}
+```
+
+By default, rates are pulled from the [CoinCap API](https://docs.coincap.io/?version=latest), which provides over 200 fiat and crypto currency prices. A `PaymentError.ExternalRateUnavailable` will be thrown if the exchange rate between the source and destination asset is unavailable from the API or wasn't provided in `prices`.
+
+##### `getExpiry`
+
+> _Optional_: `(destination: string) => Date`
+
+Callback function to set the expiration timestamp of each ILP Prepare packet. By default, the expiration is set to 30 seconds in the future.
+
+#### `Quote`
+
+TODO
+
+#### `Receipt`
+
+Outcome of the payment.
+
+##### `error`
+
+> `PaymentError` or `undefined`
+
+Type of error if the payment encountered with an error.
+
+##### `amountSent`
+
+> [`BigNumber`](https://mikemcl.github.io/bignumber.js/)
+
+Amount sent and fulfilled, in normal units of the source asset.
+
+##### `amountInFlight`
+
+> [`BigNumber`](https://mikemcl.github.io/bignumber.js/)
+
+Amount in-flight and yet to be fulfilled or rejected, in normal units of the source asset. (This should always be 0 since no receipt is returned until all pending requests complete).
```

Comment:
Whats the rational of having this if it should always be 0? Are there edge cases where the receipt will return whilst still having in-flight packets? If so why does that occur and should we document what/how the user should handle these cases.

Reply:
I _think_ the only case would be in there's some error in the logic somewhere. However, I'm thinking about emitting some kind of per-packet "receipt" so consumers could make payments more durable if the sender crashed -- I'll revise this after I figure out what to do with that functionality.